RevenueCatUI/CustomerCenter/ViewModels/CustomerCenterViewModel.swift

```diff
@@ -194,16 +197,15 @@ import Foundation
             try await self.purchasesProvider.syncPurchases() :
             try await purchasesProvider.customerInfo(fetchPolicy: .fetchCurrent)
 
-            try await self.loadPurchases(customerInfo: customerInfo)
             try await self.loadCustomerCenterConfig()
+            try await self.loadPurchases(customerInfo: customerInfo, changePlans: configuration?.changePlan ?? [])
```

Comment:
hmm maybe `loadCustomerCenterConfig` should return the `configuration`? I find it a bit strange that we load the configuration and then load the configuration viewModel's property